.github/workflows/lint.yml

```diff
@@ -22,9 +22,9 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: "14"
+          node-version-file: .tool-versions
```

Comment:
This is only required in CI, and should not affect devs

Reply:
that is fine, I will bump it to the latest LTS version